### PR TITLE
[CF-217] Fix CheckUsersAvailabilityOnSrcAndDst

### PR DESCRIPTION
**Pull request by antonf**

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.actions.check_users
cloudferrylib/os/actions/check_users.py:48: [W0141(bad-builtin), CheckUsersAvailabilityOnSrcAndDst.run] Used builtin function 'map'. Using a list comprehension can be clearer.
cloudferrylib/os/actions/check_users.py:50: [W0141(bad-builtin), CheckUsersAvailabilityOnSrcAndDst.run] Used builtin function 'map'. Using a list comprehension can be clearer.

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: cloudferrylib/os/actions/check_users.py [hunk 1]
```diff
@@ -44,15 +44,19 @@ def run(self, **kwargs):
...
-        users_missing_on_dst = src_user_names - dst_user_names
+        users_missing_on_dst = \
+            set(src_user_names.keys()) - set(dst_user_names.keys())
 
         if users_missing_on_dst:
             msg = "{n} missing users on destination: {users}".format(
                 n=len(users_missing_on_dst),
-                users=", ".join(users_missing_on_dst))
+                users=", ".join(users_missing_on_dst[key]
+                                for key in users_missing_on_dst))
```
Comment (roman-verchikov) [inline cloudferrylib/os/actions/check_users.py]: `sorted(users_missing_on_dst)` would be easier to read

**Comment by archyufa** [comment]
Failed on staging

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.actions.check_users
cloudferrylib/os/actions/check_users.py:48: [W0141(bad-builtin), CheckUsersAvailabilityOnSrcAndDst.run] Used builtin function 'map'. Using a list comprehension can be clearer.
cloudferrylib/os/actions/check_users.py:50: [W0141(bad-builtin), CheckUsersAvailabilityOnSrcAndDst.run] Used builtin function 'map'. Using a list comprehension can be clearer.

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by archyufa** [comment]
+1 tested on staging can be merged

**Comment by roman-verchikov** [comment]
+1, can be merged

**Pull request merged**